react/finger-exercises/ecmascript/01 - spread/index.js

```diff
@@ -1,10 +1,24 @@
-import { isArray } from './utils';
+import isArray from './utils';
 
-export function min() {
-  isArray();
-  return Math.min();
+export function min(...args) {
+  if (args.length === 0) {
```

Comment:
You can check this by negating the length 
`if (!args.length) {`

Reply:
negating the length will change the logic of the function, what about using `if(!!args.length){ ...`